### DISCUSSION TITLE
Managing public perception

### DISCUSSION THREAD

**Discussion by gregsdennis** [json-schema-org/community#463] [Vision and direction]
I recently had communicated to me some external perceptions about the JSON Schema project that I think we as a team need to be aware of:  since the primary active workers of the project are employed by Postman, JSON Schema is now perceived to be a Postman project.

An outcome of this is that any perceptions of how Postman is doing reflects on how this project is doing.  When it's perceived that there may be turmoil at the company, confidence in this project's future is diminished.  It's public knowledge that Kin is no longer leading Postman's Open Technologies section (his LinkedIn profile shows he has a new role now), and a couple other rumors were expressed to me that I won't entertain here.

I consider this a problem for JSON Schema, not just for the employees of Postman, which is why I'm bringing this up here.  If there's any aspect of this that also needs to be addressed within Postman, I'm happy to take that there.

Thinking about this, it's the fact that Postman is paying some (majority) of us that's driving this perception.  It doesn't matter

- that literally no one at Postman, especially leadership, have given any of us (to my knowledge, certainly not me) any kind of direction that could influence the project, or
- that the project is officially under OpenJS ownership.

I, for one, am grateful that Postman has provided for this project by providing a living wage for several of us.  My wish is that everyone to have such sponsorship from multiple companies.  However, it seems that being completely open about Postman's actual influence (again, basically none) doesn't change that perception because the people saying it (us) are Postman employees.

This was one of my initial concerns before joining Postman.  Now that I've spent a year there and have experienced complete autonomy, I'm less worried about Postman taking over.  (We provide weekly status updates in Postman's Slack, and I also expect that Ben has to provide some kind of report of progress in order to keep their interest.)

This project's openness and autonomy are important to me personally.  These aspects are hallmarks of what open source (and specifications) should be.  I realize that we do have a lot more contributors lately, but they've only operated by helping develop tools and the website.  None of these new people are contributing in discussions that affect spec development.  I want more people involved in spec development.  (Especially, I'd like Karen and Austin more involved again.)

Getting more people involved in spec development (whether in discussions or even authorship), to me, is the only way that we'll be able to combat and perhaps reverse this perception.  This isn't a Postman project.  They didn't buy us.  If Postman backed out and we all had to go find new jobs, I'd still be here.  But the _perception_ is that they did buy the project, and we need to change that.

**Comment by Relequestual** [comment]
Two things immediatly come to mind on this.
Firslty, when we do establish the TSC (Which should be pretty soon), I'm going to propose @mwadams joins as co-chair. He is the CTO and co-found of Endjin. Given he has been involved occasionally over the past few months, and is willing to commit time and effort to running the TSC, I'm hopeful that such an addition will send a good signal in addition to being very beneficial for JSON Schema.
Secondly, when we complete our OpenJS Foundation onboarding, there will be some fanfair. I'm guessing we can make suggestions or requests on messages we want to include in such fanfair. One of which I think should be we are looking for additional TSC membership. We should probably discuss what qualities and expereince we are looking for in TSC members.

**Reply by gregsdennis** [reply]
This is the first I'm hearing of this.  I think this will definitely help.

**Comment by jdesrosiers** [comment]
I think there's two thing people might be concerned about. I'm not sure which one we're talking about here (or both).

**Postman "controlling" JSON Schema**

If people are concerned about Postman controlling JSON Schema in any way, they couldn't be more wrong. As promised, Postman has left us completely alone to do our thing. We've taken steps to make sure Postman can't own JSON Schema by joining OpenJS. If people actually believe Postman controls JSON Schema, they are just making assumptions or listening to rumors. If we really think people have this misunderstanding, we should address it, but I'm not yet convinced this is the case.

**JSON Schema's health being tied to the health of Postman**

If people are worried about this, they're kind of right to be worried. We can do a lot more a lot quicker with funding from Postman. However, all of us that are employed by Postman were doing this work before Postman got involved and will continue to do this work if Postman decides to stop supporting us. Before Postman hired us, I remember having conversations with people who were concerned about adopting JSON Schema because it wasn't backed by a company. These days, that's what a lot of people expect of a stable project they can trust to be around in the future (Example: Meta backing React and GraphQL). Overall, I expect Postman's support to give confidence in the project.

> When it's perceived that there may be turmoil at the company, confidence in this project's future is diminished.

I haven't been paying much attention to social media these days, but is there really a perception that Postman is in turmoil right now? People get burned out and change jobs all the time, especially in tech. It doesn't mean the company is in trouble. This will pass.

---

I'm skeptical that this is truly an urgent problem. I personally have seen no signs at all of anyone having any opinion (good or bad) of Postman's support of JSON Schema. I doubt most people even know, let alone care, that we employed by Postman. Clearly there's a lot that isn't said in this post and maybe some of that might change my mind. Maybe those details are best shared in private.

**Reply by jdesrosiers** [reply]
That's not to say that I don't think we need to continue to try to mitigate both of those risks. I just mean I'm not yet concerned that a widespread perception problem exists.
